Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.9
```

Comment:
@icanzilb Can we go down as couple of minor versions to 5.6 or something like that maybe to include more environments?

Reply:
Yeah, 5.4 seems to be the earliest one supporting resources

Reply:
https://github.com/onekiloparsec/KPCTabsControl/pull/43/commits/01c5d0ad25c9af218b24d7ef1d4f32acd3300c84